docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  mysql_db:
+    container_name: mysql_db
+    restart: always
+    image: "mysql:8"
+    command: --default-authentication-plugin=mysql_native_password
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_HOST=localhost
+      - MYSQL_USER=user
+      - MYSQL_DB=mysql_db
+      - MYSQL_PASSWORD=root
+      - MYSQL_PORT=3306
+      - MYSQL_ROOT_PASSWORD=root
```

Comment:
really?  we push to github environment variables? have you heard anything about security? do you know why I sad no real envs in github?